portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,14 +44,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     // Prepares results and sorts them in descending order.
     Query query = new Query("entry").addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
+    // TODO(aabundis): Find efficient way other than clearing arrayList and then filling up again.
```

Comment:
I'm glad you're thinking about this. Keep thinking on it, perhaps we can chat through it a little in our 1:1 :)

Reply:
Will do!